sjb/config/test_cases/test_branch_origin_extended_conformance_crio_rpm.yml

```diff
@@ -38,6 +38,35 @@ extensions:
         name = OpenShift Ansible Release from Local Source
         EOR
         sudo cp ./openshift-ansible-local-release.repo /etc/yum.repos.d
+    - type: "script"
+      title: "turn off unnecessary RPM repos"
+      timeout: 300
+      script: |-
+        sudo yum-config-manager --disable centos-paas-sig-openshift-origin\*-rpms
+        sudo yum-config-manager --disable origin-deps-rhel7\* || true
+        sudo yum-config-manager --disable rhel-7-server-ose\* || true
+
+        branch="${PULL_BASE_REF:-"master"}"
+        case "${branch}" in
+          master )
+            # 3.10
+            major=3
+            minor=10
+            ;;
+          release-* )
+            major="$( echo "${branch}" | grep -Po "(?<=release-)[0-9](?=\.[0-9])" )"
+            minor="$( echo "${branch}" | grep -Po "(?<=release-[0-9]\.)[0-9]" )"
+            ;;
+          enterprise-* )
+            major="$( echo "${branch}" | grep -Po "(?<=release-)[0-9](?=\.[0-9])" )"
+            minor="$( echo "${branch}" | grep -Po "(?<=release-[0-9]\.)[0-9]" )"
+            ;;
+        esac
+        previous="$(( minor -1))"
+        sudo yum-config-manager --enable "rhel-7-server-ose-${major}.${minor}"
```

Comment:
@sdodson you know more than I: Are all the above complications necessary?  Couldn't we simply rely on yum/rpm to always pick the latest version available.  In other words, just:  ``yum-config-manager --enable "rhel-7-server-*"`` or does that cause problems?

Reply:
We use the same jobs to install 3.9, 3.10, and future releases so we need to make sure we enable only the correct repos.  AFAIK cri-o 1.10 is not guaranteed to be compatible with Origin 3.9

Reply:
OIC...thx for the explanation.  It seems like a lot of extra maintenance work, given the total number of jobs here (not just CRI-O).  Is this section at all common with other jobs?  (Just trying to think if I could help out writing some shared script or table or something)